Adjust Margins and Font Properties for Meeting Spaces Card

greenteaisgreat opened the pull request:
# Pull Request

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

---

This addresses some minor styling issues for [#21](https://github.com/orgs/APL-Innovation-Lab/projects/1/views/6?pane=issue&itemId=69438535)

Changes include:
- Adjust the "Meeting Spaces" heading text to be the correct font and font-size, per the [Figma Flow Map](https://www.figma.com/design/DMjN6uhMLOw7VZO3t4eqjd/APL-Meeting-Room?node-id=1225-43028&node-type=SYMBOL&t=b1U1b5tCjfcPtdGX-0).
- Adjust the left and right margins for the main content area
- Remove text-decorations for `<Breadcrumbs />` component links

**Type of change:**

-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] Refactor (non-breaking change which improves existing functionality)
-   [ ] Test (add missing tests or correct existing tests)
-   [ ] Chore (update to build tasks, package manager configs, etc; no production code change)
-   [ ] Documentation (update project documentation)

## Screenshots

Please take screenshots of the screens that are affected as a part of your pull request on both iOS and Android. Drag images into this PR and GitHub will generate an image link.

---
Max-Width
![desktop](https://github.com/user-attachments/assets/f57146d0-c72b-4750-93a4-8c6c83c7736d)

Min-Width
![mobile](https://github.com/user-attachments/assets/bbc81a0a-b8aa-4e4c-a532-6ba61a97d24a)

julielaursen commented:
is the pull request template not working here?
Can we add screenshots to the PR?
Can the correct screen on the figma flow map be selected or can we only link to the entire figma? It'd be great if it zoomed into the page we're modifying

greenteaisgreat commented:
> is the pull request template not working here? 

No, I'm just a dumb-dumb that forgot about that. I updated the PR to follow the provided template. Thanks for reminding about that, Julie! I also changed the Figma link to be more direct in the area of interest.

markmals reviewed (approved):
LGTM
